fairworkflows/fairstep.py

```diff
@@ -494,16 +494,26 @@ def _extract_outputs_from_function(func, additional_params) -> List[FairVariable
                          'FAIR step functions MUST have type hinting, '
                          'see https://docs.python.org/3/library/typing.html')
     if _is_generic_tuple(return_annotation):
+        return_sem_types = additional_params.get('out')
```

Comment:
Maybe extract a constant for 'out'?

Reply:
Good idea. I've added a constant for this to the ```fairworkflows.config```